cosmwasm/deploy-contract.js

```diff
@@ -360,8 +360,8 @@ const instantiate = (client, wallet, chainName, config, options) => {
     const initMsg = makeInstantiateMsg(contractName, chainName, config);
     return instantiateContract(client, wallet, initMsg, config, options).then((contractAddress) => {
         if (chainConfig) {
-            contractConfig[chainConfig.id] = {
-                ...contractConfig[chainConfig.id],
+            contractConfig[chainConfig.axelarId] = {
```

Comment:
@jcs47 do we refer to `id` in the wasm deployment action? can we change it to axelarId there as well if so?

Reply:
IIRC I already fixed something related to this in the orchestration. But I am not sure if this PR will break anything else. I'll keep an eye on the pipelines.